tests/integration/test.attachments.js

```diff
@@ -1749,7 +1749,7 @@ adapters.forEach(function (adapter) {
                 cache: true,
                 binary: true
               }, function (err, res) {
-                if (err && err.status === 405) {
+                if (err && err.status === 500) {
```

Comment:
As per the comment in the test, this check only to skip the tests in firefox, we previously had done https://github.com/pouchdb/pouchdb/blob/master/lib/deps/ajax-browser.js#L80 to always force a 405 which was causing various ways xhr could fail so misreport as cors problems, afaik it should affect blob detection

We are testing in fx 31 in travis which is older than we support anyway